ext/opentelemetry-ext-jaeger/README.rst

```diff
@@ -0,0 +1,67 @@
+OpenTelemetry Jaeger Exporter
+=============================
+
+Installation
+------------
+
+::
+
+     pip install opentelemetry-ext-jaeger
+
+
+Usage
+-----
+
+The **OpenTelemetry Jaeger Exporter** allows to export `OpenTelemetry`_ traces to `Jaeger`_.
+This exporter always send traces to the configured agent using Thrift compact protocol over UDP.
+An optional collector can be configured, in this case Thrift binary protocol over HTTP is used.
+gRPC is still not supported by this implementation.
+
+
+.. _Jaeger: https://www.jaegertracing.io/
+.. _OpenTelemetry: https://github.com/opentelemetry/opentelemetry-python/
```

Comment:
FYI: this link is incorrect. it should be open-telemetry for now.

Reply:
Thanks!, I think I'm contributor number one with more typos done...
Will be solved in https://github.com/open-telemetry/opentelemetry-python/pull/289.